airbyte-integrations/connectors/destination-postgres/src/main/resources/spec.json

```diff
@@ -218,7 +218,7 @@
       },
       "use_1s1t_format": {
         "type": "boolean",
-        "description": "(Early Access) Use <a href=\"https://docs.airbyte.com/understanding-airbyte/typing-deduping\" target=\"_blank\">Destinations V2</a>.",
+        "description": "(Early Access) Use <a href=\"https://docs.airbyte.com/using-airbyte/core-concepts/typing-deduping\" target=\"_blank\">Destinations V2</a>.",
```

Comment:
Not a strong opinion but probably not worth publishing new versions of Postgres + Postgres Strict Encrypt just for this; the next versions can just pick this up.